content/blog/2022/10/envvar-policy.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Enforcing compliance in your environment variables
+authors:
+- José Guilherme Vanz
+date: 2022-11-01
+---
+
+We're glad to present to Kubewarden users the new [environment-variable-policy](https://github.com/kubewarden/environment-variable-policy) policy.
+It's able to inspect containers, init containers, and ephemeral containers, and restricts
+their usage of environment variables by checking their names and values defined under the
+containers' [`env[*]`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#envvar-v1-core).
+As always, the policy can be found in [Github](https://github.com/kubewarden/environment-variable-policy)
+and all the  artifacts, including the BOM files, are signed with Sigstore
+
+### What is so useful about the new environment-variable policy?
+
+The recent added policy https://github.com/kubewarden/env-variable-secrets-scanner-policy allow users
+to check if there are some leaked sensitive data in the environment variables. However, until the
+environment-variable-policy there were no way to check other environment variables aspects.
+This is fixed now. In this new policy users can validate which variables name and values their
```

Comment:
I don't **personally** like the "this is fixed now" and the way we get there. I would rephrase it be something like:

```
This policy complements the recently released <link to secret scanner blog post> policy. Both policies focus on validating the environment values provided to Kubernetes Pod objects.

In this new policy, users can validate...
```